src/openssl/algorithm.cr

```diff
@@ -0,0 +1,35 @@
+require "openssl"
+
+module OpenSSL
+  enum Algorithm
+    # This enum indicates the possible hash algorithms for *OpenSSL* digest operations.
```

Comment:
This needs to be put outside of the enum so it's visible in the documentation:
```cr
# Documentation
enum Enum
  ...
end
```
Also `This enum` can be excluded, it's clear what `Algorithm` is.

Reply:
Thanks.

Reply:
This one was fixed as suggested